lib/ansible/module_utils/urls.py

```diff
@@ -1123,6 +1123,7 @@ def open(self, method, url, data=None, headers=None, use_proxy=None,
         :kwarg unix_socket: (optional) String of file system path to unix socket file to use when establishing
             connection to the provided url
         :kwarg ca_path: (optional) String of file system path to CA cert bundle to use
+        :kawrg unredirected_headers: (optional) A list of headers to not attach on a redirected request
```

Comment:
```suggestion
        :kwarg unredirected_headers: (optional) A list of headers to not attach on a redirected request
```

Reply:
We haven't done it in the past, but I think we should start adding version_added to these docs

Reply:
Makes sense, will do.